<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SitesDialog</class>
 <widget class="QWidget" name="SitesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Liste des sites d'activité</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Liste des sites</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>421</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
         <property name="buddy">
          <cstring>nameEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit">
         <property name="minimumSize">
          <size>
           <width>431</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Adresse</string>
         </property>
         <property name="buddy">
          <cstring>textEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="minimumSize">
          <size>
           <width>431</width>
           <height>131</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Code Postal</string>
         </property>
         <property name="buddy">
          <cstring>cpEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cpEdit">
         <property name="minimumSize">
          <size>
           <width>431</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Ville</string>
         </property>
         <property name="buddy">
          <cstring>townEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="townEdit">
         <property name="minimumSize">
          <size>
           <width>431</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Téléphone</string>
         </property>
         <property name="buddy">
          <cstring>phoneEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="phoneEdit">
         <property name="minimumSize">
          <size>
           <width>201</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Fax</string>
         </property>
         <property name="buddy">
          <cstring>faxEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="faxEdit">
         <property name="minimumSize">
          <size>
           <width>201</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Mail</string>
         </property>
         <property name="buddy">
          <cstring>mailEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mailEdit">
         <property name="minimumSize">
          <size>
           <width>201</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="recordButton">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>27</height>
          </size>
         </property>
         <property name="text">
          <string>Enregi&amp;strer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="preferedButton">
         <property name="text">
          <string>Choisir 
comme préférentiel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modifyButton">
         <property name="text">
          <string>&amp;Modifier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>27</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Quitter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delButton">
       <property name="text">
        <string>Del</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>409</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
